definitions.go

```diff
@@ -20,14 +24,21 @@ func GetResourceDefinitionTasks(tasks chan<- Task, runner Runner, projects, reso
 // definition for the given resource in the given project.
 func ResourceDefinition(r Runner, project, resource string) Task {
```

Comment:
Technically `resource` could be the empty string too.
I prefer the "logical" order that one normally specify from broader to narrower: project -> resource type -> resource name, project -> pod -> pod name -> container name, ...

I don't see much value in wrapping with another function. One can still pass any resource, even if it is not "cluster scoped".

E.g. calling `PlatformResourceDefinition(r, "services")` will return all services in the current project -- but the "current project" is out of control of the dump tool, I'd say that would be a mistake to make such call in the context of the tool.

In the end of the day `ResourceDefinition` is just a wrapper to `oc [-n PROJ] get [KIND/NAME] [...]`, it could be called `OcGet` :D